src/components/Categories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Categories = () => { //eslint-disable-line
+
+  return (
+    <div className="category">
+      <h1 className="categoriesHeader">Categories</h1>
```

Comment:
- [ ] I think it would be better if you add a button here called check status as required so kindly add a button here 💯🔥🚀
![image](https://user-images.githubusercontent.com/58553711/200139127-72c233ab-1f8e-4cbf-8654-a6ed2a044af2.png)

Reply:
I will apply the recommendations